build/posix.sh

```diff
@@ -107,6 +107,8 @@ CURL="curl --silent --location --retry 3 --retry-max-time 30"
 if [ "$DARWIN" = true ]; then
   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \
     | sh -s -- -y --no-modify-path --profile minimal
+  rustup default nightly
```

Comment:
Instead of doing `rustup default nightly`, you can also use `--default-toolchain nightly` during installation. 
```suggestion
  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \
    | sh -s -- -y --no-modify-path --profile minimal --default-toolchain nightly
```